capstone-frontend/src/components/BookDescriptionOverlay.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Popover, OverlayTrigger } from 'react-bootstrap';
+
+const popoutDescription = (book) => {
+  let description = book.description;
+  if (description === undefined) {
+    description = "";
+  }
+  if (description.length > 500) {
+    description = description.substring(0, 500).concat('...');
+  }
+
+  return (
+    <Popover>
+      <Popover.Title as='h3'>
+        <a className='center-horizontal'
```

Comment:
I think we should use a react-router `Link` here instead of an anchor tag. In `App.jsx` we define the `Route` for "/bookpage/:id" and I think it would be cleaner to pair that Route with a corresponding Link here.

Reply:
Added!